src/components/App.js

```diff
@@ -1,15 +1,16 @@
 import React from 'react';
+import { FormattedMessage, injectIntl } from 'react-intl';
 import { Link } from 'react-router-dom';
 
-const App = () => (
+const App = ({ intl }) => (
   <div className="App">
     <header className="App-header">
       <p>
-        Edit <code>src/App.js</code> and save to reload.
+        <FormattedMessage id="home.hello" defaultMessage="Terve" />
```

Comment:
I see a lot dangers with using `defaultMessage` 🤔 and wouldn't like to have them.
- It makes hard to spot missing translations
- When the translations changes, the default value doesn't get updated (causes more work to maintain them properly)
- Can lose context over time
- Adding arbitrary content which never should not be visible.